doc/releases/release-notes-3.2.rst

```diff
@@ -5,6 +5,17 @@
 Zephyr 3.2.0 (Working Draft)
 ############################
 
+We are pleased to announce the release of Zephyr version 3.2.0.
+
+Major enhancements with this release include:
```

Comment:
I think this deserves a mention:
https://github.com/zephyrproject-rtos/zephyr/blame/main/doc/releases/release-notes-3.2.rst#L1333-L1335

Reply:
Oh yeah, I'll add it.